### PR TITLE
Malum compat and a bunch of other stuff

### PR DESCRIPTION
**Pull request by f-raZ0R**
This WAS going to be a malum compat PR, except i got sidetracked about halfway through and implemented pure zinc and some other stuff too.
I forgot some of the stuff I even implemented in this tbh

Anyways I now have crippling burnout and am probably going to take a break from mod development for a while after this.

YES I KNOW THERES NO GUIDEBOOK STUFF, i was planning on adding stuff to MALUM's guidebook, but unfortunately their API doesn't let me do this easily so i gave up on that :(

### CODE REVIEW

File: src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java [hunk 1]
```diff
@@ -37,10 +44,23 @@ public int getMaxPower(int level) {
 	public int getMaxLevel() {
 		return SpectrumCommon.CONFIG.CloversFavorMaxLevel;
 	}
+
+	@Override
+	public boolean isAcceptableItem(ItemStack stack) {
+		var item = stack.getItem();
+		if(SpectrumIntegrationPacks.isIntegrationPackActive(SpectrumIntegrationPacks.MALUM_ID) &&
+				item instanceof MalumScytheItem)
```
Comment (DaFuqs) [inline src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java:52]: Does this run in an instance without Malum installed?
Mentioning `MalumScytheItem` here may throw a class load error.

(same in later cases. But I'll not annotate every single one for now)

File: src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java [hunk 2]
```diff
@@ -37,10 +44,23 @@ public int getMaxPower(int level) {
...
+		}
+
+		return super.isAcceptableItem(stack);
+	}
 	
 	@Override
 	public boolean canAccept(Enchantment other) {
-		return super.canAccept(other) && other != Enchantments.LOOTING;
+		return super.canAccept(other) && other != Enchantments.LOOTING &&
+				!(SpectrumIntegrationPacks.isIntegrationPackActive(SpectrumIntegrationPacks.MALUM_ID) && other == EnchantmentRegistry.SPIRIT_PLUNDER.get());
```
Comment (DaFuqs) [inline src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java:63]: Does this run in an instance without Malum installed?
Mentioning `EnchantmentRegistry.SPIRIT_PLUNDER` may throw a class load error.

File: src/main/resources/assets/spectrum/lang/en_us.json [hunk 3]
```diff
@@ -4096,7 +4100,7 @@
   "enchantment.spectrum.razing.desc": "Higher mining speed the harder a block is to break",
   "enchantment.spectrum.resonance": "Resonance",
   "enchantment.spectrum.resonance.desc": "Uses gentle vibrations to allow some blocks to either drop themselves or preserve special properties",
-  "enchantment.spectrum.serendipity_reel": "Serendipity_reel",
+  "enchantment.spectrum.serendipity_reel": "Serendipity Reel",
```
Comment (DaFuqs) [inline src/main/resources/assets/spectrum/lang/en_us.json:4103]: Good catch!

File: src/main/resources/data/malum/tags/blocks/cursedworkaroundjank.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+please remove this as soon as it can be done this is cursed voodoo
```
Comment (DaFuqs) [inline src/main/resources/data/malum/tags/blocks/cursedworkaroundjank.txt:1]: Why have decay away in the tag in the first place?
grass/flowers behave the same way, when the grass_block below them is removed

File: src/main/resources/data/malum/tags/items/spirit_crystals.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "values": [
+    "malum:sacred_spirit",
+    "malum:wicked_spirit",
+    "malum:arcane_spirit",
+    "malum:eldritch_spirit",
+    "malum:aerial_spirit",
+    "malum:aqueous_spirit",
+    "malum:earthen_spirit",
+    "malum:infernal_spirit"
```
Comment (DaFuqs) [inline src/main/resources/data/malum/tags/items/spirit_crystals.json:10]: Adding malum items to a malum tag? o.O

File: src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json [hunk 6]
```diff
@@ -1,20 +1,37 @@
...
         "advancement_identifier": "spectrum:hidden/collect_vanilla/ender_eye"
       }
     },
+    "collected_warp_flux": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "malum:warp_flux"
```
Comment (DaFuqs) [inline src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json]: doesn't this throw an error when malum is not loaded, making the whole file break?

File: src/main/resources/data/spectrum/recipes/mod_integration/malum/pedestal/alt_bag_of_holding.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  "magenta": 4,
+  "yellow": 0,
+  "cyan": 4,
+  "black": 0,
+  "white": 0,
+  "experience": 2.0,
+  "pattern": [
+    "OOO",
+    "TWT",
+    "OOO"
```
Comment (DaFuqs) [inline src/main/resources/data/spectrum/recipes/mod_integration/malum/pedestal/alt_bag_of_holding.json:14]: we should start writing shaped recipes that read "OWO"

File: src/main/resources/data/spectrum/recipes/mod_integration/the_grande_spectacular_mod_compat_double_feature/malum_create/anvil_crushing/brilliance_crushing.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  "result": {
+    "item": "malum:crushed_brilliance"
+  },
+  "particleEffectIdentifier": "enchanted_hit",
+  "soundEventIdentifier": "block.metal.break",
+  "fabric:load_conditions": [
+    {
+      "condition": "fabric:all_mods_loaded",
+      "values": [
+        "create",
```
Comment (DaFuqs) [inline src/main/resources/data/spectrum/recipes/mod_integration/the_grande_spectacular_mod_compat_double_feature/malum_create/anvil_crushing/brilliance_crushing.json:17]: wait, why does this require create to be present? Is crushed_brilliance a malum<->create compat item?

**Review by DaFuqs — CHANGES_REQUESTED** [review]

File: src/main/resources/data/spectrum/recipes/mod_integration/the_grande_spectacular_mod_compat_double_feature/malum_create/anvil_crushing/brilliance_crushing.json (hunk 8 above)
Comment (f-raZ0R) [reply src/main/resources/data/spectrum/recipes/mod_integration/the_grande_spectacular_mod_compat_double_feature/malum_create/anvil_crushing/brilliance_crushing.json:17]: Yep, this and crushed soulstone are create compat items for crushing malum ores in a create crushing wheel.

File: src/main/resources/data/malum/tags/blocks/cursedworkaroundjank.txt (hunk 4 above)
Comment (f-raZ0R) [reply src/main/resources/data/malum/tags/blocks/cursedworkaroundjank.txt:1]: I don't really _like_ the outcome of using decay away on blight causing all of the blight plants to pop off and drop as items...

File: src/main/resources/data/malum/tags/items/spirit_crystals.json (hunk 5 above)
Comment (f-raZ0R) [reply src/main/resources/data/malum/tags/items/spirit_crystals.json:10]: Yeah uhh.. i was kinda tired when I wrote this, this should probably not be in malum's namespace, but it turns out that malum just.. doesn't **have** a tag for spirit crystals!

File: src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java (hunk 1 above)
Comment (f-raZ0R) [reply src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java:52]: it runs perfectly fine, actually!

File: src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java (hunk 2 above)
Comment (f-raZ0R) [reply src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java:63]: it runs perfectly fine, actually!

File: src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json (hunk 6 above)
Comment (f-raZ0R) [reply src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json]: .... oh dear yeah it does. how would I fix this then?

Comment (f-raZ0R) [reply src/main/resources/data/spectrum/advancements/unlocks/items/bag_of_holding.json]: Fixed

**Pull request merged**